generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.ts.ejs

```diff
@@ -181,28 +181,28 @@ export default (state: AdministrationState = initialState, action): Administrati
 <%_ if (applicationType === 'gateway' && serviceDiscoveryType) { _%>
 export const gatewayRoutes = () => ({
   type: ACTION_TYPES.FETCH_GATEWAY_ROUTE,
-  payload: axios.get('/api/gateway/routes')
+  payload: axios.get('api/gateway/routes')
```

Comment:
is the `/` automatically added? I didnt check earlier

Reply:
Having the path start with `/` will make a request to the base domain/IP instead of the relative path.  

For example, app is running on `http://jhipster.com/sample-app/` 
- Requesting `/api/account` routes to `http://jhipster.com/api/account`
- Requesting `api/account` routes to `http://jhipster.com/sample-app/api/account`